sdk/storage/azure_storage_blob/src/generated/clients/blob_client.rs

```diff
@@ -66,111 +90,1375 @@ impl BlobClient {
         &self.endpoint
     }
 
-    /// Returns a new instance of BlobAppendBlobClient.
+    /// The Abort Copy From URL operation aborts a pending Copy From URL operation, and leaves a destination blob with zero length
+    /// and full metadata.
     ///
     /// # Arguments
     ///
-    /// * `container_name` - The name of the container.
-    /// * `blob` - The name of the blob.
-    pub fn get_blob_append_blob_client(
+    /// * `copy_id` - The copy identifier provided in the x-ms-copy-id header of the original Copy Blob operation.
+    /// * `options` - Optional parameters for the request.
+    pub async fn abort_copy_from_url(
         &self,
-        container_name: String,
-        blob: String,
-    ) -> BlobAppendBlobClient {
-        BlobAppendBlobClient {
-            blob,
-            container_name,
-            endpoint: self.endpoint.clone(),
-            pipeline: self.pipeline.clone(),
-            version: self.version.clone(),
+        copy_id: &str,
+        options: Option<BlobClientAbortCopyFromUrlOptions<'_>>,
+    ) -> Result<Response<BlobClientAbortCopyFromUrlResult>> {
```

Comment:
Making this comment here but this comment applies broadly to the whole new generated code-- each `Result` returned from these generated REST APIs have a unique `Result` that is a combo of <operation> + <Result> i.e. `BlobCLientAbortCopyFromUrlResult`. They no longer return a general `Response`.